test/services/db-export/send-to-s3-bucket.service.test.js

```diff
@@ -0,0 +1,94 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+const Sinon = require('sinon')
+
+const { describe, it, beforeEach, afterEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Test helpers
+const path = require('path')
+
+// Things we need to stub
+const { S3Client, PutObjectCommand } = require('@aws-sdk/client-s3')
+
+// Thing under test
+const SendToS3BucketService = require('../../../app/services/db-export/send-to-s3-bucket.service')
+
+describe('Send to S3 bucket service', () => {
+  let s3Stub
+  let notifierStub
+
+  beforeEach(() => {
+    notifierStub = { omg: Sinon.stub(), omfg: Sinon.stub() }
+    global.GlobalNotifier = notifierStub
+  })
+
+  afterEach(() => {
+    Sinon.restore()
+    delete global.GlobalNotifier
+  })
+
+  describe('when successful', () => {
+    beforeEach(() => {
+      // Stub the S3 Client's send method, which is used to run the 'put object' command
+      s3Stub = Sinon.stub(S3Client.prototype, 'send')
+    })
+
+    const filePath = 'test/fixtures/compress-files.service.csv'
+    const fileName = path.basename(filePath)
+
+    it('uploads a file to the S3 bucket', async () => {
+      const result = await SendToS3BucketService.go(filePath)
+
+      // Test that the S3 Client was called once
+      expect(s3Stub.calledOnce).to.be.true()
+
+      // Get the first call and test that it was called with PutObjectCommand
+      const calledCommand = s3Stub.getCall(0).firstArg
+      expect(calledCommand instanceof PutObjectCommand).to.be.true()
```

Comment:
There's an [`instanceof()` assertion](https://hapi.dev/module/code/api/?v=9.0.3#instanceoftype) we can use here:
```suggestion
      expect(calledCommand).to.be.an.instanceof(PutObjectCommand)
```